ElementX/Sources/Screens/ManageRoomMemberSheet/ManageRoomMemberSheetModels.swift

```diff
@@ -24,10 +50,32 @@ struct ManageRoomMemberSheetViewStateBindings {
 enum ManageRoomMemberSheetViewAlertType {
     case kick
     case ban
+    case unban
 }
 
 enum ManageRoomMemberSheetViewAction {
     case kick
     case ban
+    case unban
     case displayDetails
 }
+
+enum ManageRoomMemberDetails {
+    case memberDetails(roomMember: RoomMemberDetails)
+    case senderDetails(sender: TimelineItemSender)
+    
+    var id: String {
+        switch self {
+        case let .memberDetails(roomMember):
+            return roomMember.id
+        case let .senderDetails(sender):
+            return sender.id
+        }
+    }
```

Comment:
Can we use implicit returns here?
```suggestion
    var id: String {
        switch self {
        case let .memberDetails(roomMember):
           roomMember.id
        case let .senderDetails(sender):
           sender.id
        }
    }
```